src/app/app.component.html

```diff
@@ -1,4 +1,8 @@
 <div class="product-container">
-  <app-product></app-product>
+    <app-product></app-product>
 </div>
 <app-card></app-card>
+@for (word of texts; track word) {
```

Comment:
```suggestion
@for (word of words; track word) {
```
Je suis très relou avec le nommage des variables :D c'est très important pour la cohérence du code